src/app/components/elements/OutgoingDelegationsTables/Pagination.jsx

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable linebreak-style */
+/* eslint react/prop-types: 0 */
+import React, { Component } from 'react';
+import shouldComponentUpdate from 'app/utils/shouldComponentUpdate';
+import { LEFT_PAGE, RIGHT_PAGE } from './constants'
+/**
+ * Helper method for creating a range of numbers
+ * range(1, 5) => [1, 2, 3, 4, 5]
+ */
+const range = (from, to, step = 1) => {
```

Comment:
Minor: if `step` is negative, the page will hang. Could add a check here.

Reply:
Fix pushed.